ddtrace/llmobs/_llmobs.py

```diff
@@ -560,7 +561,7 @@ def _tag_embedding_io(cls, span, input_documents=None, output_text=None):
                 span.set_tag_str(OUTPUT_VALUE, output_text)
             else:
                 try:
-                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text))
+                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text, default=_unserializable_default_repr))
                 except TypeError:
                     log.warning("Failed to parse output text. Output text must be JSON serializable.")
```

Comment:
Should we update this error log to something more generic like "Failed to set input/output value for X span kind"?

>"Output text must be JSON serializable."

Also I think the wording of "must" here may be confusing, since this codepath is not reached even when the I/O is not JSON serializable

Reply:
I'll update the wording, but I'd like to keep insistence on the I/O object being JSON serializable (the placeholder is just a default and should not be relied on to be normal behavior)